include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -280,12 +270,74 @@ struct GemmKernel
                     sequence<GemmPipeline::kPadM, false>{});
             }
         }();
-        auto CBlockWindow_pad = make_tile_window(
+
+        return make_tuple(a_pad_view, b_pad_view, c_pad_view);
+    }
+
+    template <typename PadView>
+    CK_TILE_DEVICE auto
+    make_gemm_tile_windows(PadView&& views, const index_t i_m, const index_t i_n) const
```

Comment:
Please use coherent function naming style. Why rvalue reference is needed?

Reply:
I'm using a few rvalue references in the code to avoid copying